zcash_keys/src/encoding.rs

```diff
@@ -256,6 +265,39 @@ pub fn decode_extended_full_viewing_key(
     bech32_decode(hrp, s, |data| ExtendedFullViewingKey::read(&data[..]).ok())
 }
 
+/// Decodes an [`ExtendedFullViewingKey`] and the [`NetworkType`] that it is intended for use with
+/// from a Bech32-encoded string.
+#[cfg(feature = "sapling")]
+pub fn decode_extfvk_with_network(
```

Comment:
I know the other functions in this file don't explicitly say in their names that they're for Sapling, but that still seems odd.